### PR TITLE
SIG: add tools sig

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Signed-off-by: zhangjinpeng1987 <zhangjinpeng@pingcap.com>

Add tools sig and planner sig.

- [x] tech-leader and other members
- [x] meeting link & meeting notes

### CODE REVIEW

File: special-interest-groups/sig-tools/roles-and-organization-management.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# SIG Tools Roles and Organization Management
```
Comment (WangXiangUSTC) [inline special-interest-groups/sig-tools/roles-and-organization-management.md]: Is this SIG used for ecosystem tools?

Comment (zhangjinpeng1987) [reply special-interest-groups/sig-tools/roles-and-organization-management.md]: Yes

**Comment by zz-jason** [comment]
@zhangjinpeng1987 description for planner SIG is merged, could you revert the changes on planner sig in this PR?

**Comment by zhangjinpeng1987** [comment]
> @zhangjinpeng1987 description for planner SIG is merged, could you revert the changes on planner sig in this PR?

@zz-jason OK

File: special-interest-groups/sig-tools/member-list.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Members and their contributions:
+
+| user                                      | num_merged_PRs | earliest_PR_merged_time | latest_PR_merged_time |
+|:-----------------------------------------:|:--------------:|:-----------------------:|:---------------------:|
+
+## Tech Leads
+
+Leads:
+
+* [GregoryIan](https://github.com/GregoryIan), [PingCAP](https://pingcap.com/en/)
```
Comment (WangXiangUSTC) [inline special-interest-groups/sig-tools/member-list.md]: @kennytm is willing to take charge of the tools-sig, please update it

Comment (zhangjinpeng1987) [reply special-interest-groups/sig-tools/member-list.md]: done

**Comment by WangXiangUSTC** [comment]
LGTM

File: special-interest-groups/sig-tools/roles-and-organization-management.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## Join and Promotion
+
+### Join
+
+The minimal requirement for contributors to join the Parser SIG is to have
+the **Active Contributor** community role in the TiDB/parser repository.
```
Comment (kennytm) [inline special-interest-groups/sig-tools/roles-and-organization-management.md]: Parser SIG???

Comment (zhangjinpeng1987) [reply special-interest-groups/sig-tools/roles-and-organization-management.md]: fixed

File: special-interest-groups/sig-tools/roles-and-organization-management.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+# SIG Tools Roles and Organization Management
+
+## Join and Promotion
+
+### Join
+
+The minimal requirement for contributors to join the Tools SIG is to have
+the **Active Contributor** community role in the [binlog](https://github.com/pingcap/tidb-binlog), [lightning](https://github.com/pingcap/tidb-lightning) and [dm](https://github.com/pingcap/dm) repositories.
```
Comment (kennytm) [inline special-interest-groups/sig-tools/roles-and-organization-management.md]: @WangXiangUSTC do we also count `tidb-tools`, and BR / CDC / Dumpling?

Comment (WangXiangUSTC) [reply special-interest-groups/sig-tools/roles-and-organization-management.md]: I think should involve these repo

Comment (zhangjinpeng1987) [reply special-interest-groups/sig-tools/roles-and-organization-management.md]: Done

File: special-interest-groups/sig-tools/README.md [hunk 5]
```diff
@@ -28,3 +28,6 @@ See [SIG Tools Member List](./member-list.md)
 * The [tidb-binlog repo](https://github.com/pingcap/tidb-binlog).
 * The [tidb-lightning repo](https://github.com/pingcap/tidb-lightning).
 * The [dm repo](https://github.com/pingcap/dm/).
+* The [backup and resotre](https://github.com/pingcap/br).
+* The [ticdc](https://github.com/pingcap/ticdc).
+* The [dumpling](https://github.com/pingcap/dumpling).
```
Comment (kennytm) [inline special-interest-groups/sig-tools/README.md]: still missing https://github.com/pingcap/tidb-tools 😄

Comment (zhangjinpeng1987) [reply special-interest-groups/sig-tools/README.md]: done

**Pull request merged**